src/quiz.js

```diff
@@ -0,0 +1,57 @@
+const INITIAL_SCORE = 0;
+const HIGH_SCORE = 1;
```

Comment:
nice use of constants, but why is this 1?

Reply:
a high score of 0 seemed really lame!